### ISSUE TITLE
Add tha possibility of have several paths to samples directory

### ISSUE THREAD

**Issue by ycollet** [LMMS/lmms#2842]
Today, we only have a samples directory in the user lmms directory.
A good thing should be to be able to add several paths for this:
lmms/samples ; ~/samples (this one is in my home directory), ... 
I know I can locate these samples via the file directory icon, but for organisation, having the samples in the same place is easier.

**Comment by mikobuntu** [comment]
I agree with this. We know that it is easy to create a sym link from lmms/samples > ~/samples.
Iirc it was/is possible to set multiple LADSPA paths in the settings using either ";" or ":" as a separator.

**Comment by lukas-w** [comment]
Consolidated into #4309.
